src/get-all-static-tiles.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const getTileUrl = require('../utils/get-tile-url');
+const getStaticTile = require('./get-static-tile');
+
+function getAllTiles(tileImgProperties, { style, accessToken }) {
+  const tiles = tileImgProperties.tiles;
+  const promises = [];
+  tiles.forEach((tile) => {
+    const tileUrl = getTileUrl({ style, accessToken }, tile.z, tile.x, tile.y);
+    // ensure our pixel offsets are associated with the tiles so we can
+    // properly blend them when stitching tiles later
+    const { px, py } = tile;
+    promises.push(getStaticTile(tileUrl, { x: px, y: py }));
+  });
+
+  return Promise.all(promises).catch((e) => {
```

Comment:
`promises` could also just be a `tiles.map`

Reply:
`tiles.map` is a great suggestion! 🙌